### ISSUE TITLE
2025-05-27, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#182]
# Today's Arxiv Papers

(1) [MineAnyBuild: Benchmarking Spatial Planning for Open-world AI Agents](http://arxiv.org/abs/2505.20148v1), 2025-05-26

(2) [FUDOKI: Discrete Flow-based Unified Understanding and Generation via Kinetic-Optimal Velocities](http://arxiv.org/abs/2505.20147v1), 2025-05-26

(3) [StructEval: Benchmarking LLMs' Capabilities to Generate Structural Outputs](http://arxiv.org/abs/2505.20139v1), 2025-05-26

(4) [Iterative Self-Incentivization Empowers Large Language Models as Agentic Searchers](http://arxiv.org/abs/2505.20128v1), 2025-05-26

(5) [Agentic AI Process Observability: Discovering Behavioral Variability](http://arxiv.org/abs/2505.20127v1), 2025-05-26

(6) [TrojanStego: Your Language Model Can Secretly Be A Steganographic Privacy Leaking Agent](http://arxiv.org/abs/2505.20118v1), 2025-05-26

(7) [Named Entity Recognition in Historical Italian: The Case of Giacomo Leopardi's Zibaldone](http://arxiv.org/abs/2505.20113v1), 2025-05-26

(8) [ResSVD: Residual Compensated SVD for Large Language Model Compression](http://arxiv.org/abs/2505.20112v1), 2025-05-26

(9) [Language-Agnostic Suicidal Risk Detection Using Large Language Models](http://arxiv.org/abs/2505.20109v1), 2025-05-26

(10) [From Data to Modeling: Fully Open-vocabulary Scene Graph Generation](http://arxiv.org/abs/2505.20106v1), 2025-05-26

(11) [Adaptive Deep Reasoning: Triggering Deep Thinking When Needed](http://arxiv.org/abs/2505.20101v1), 2025-05-26

(12) [AdaTP: Attention-Debiased Token Pruning for Video Large Language Models](http://arxiv.org/abs/2505.20100v1), 2025-05-26

(13) [Large Language Models Meet Knowledge Graphs for Question Answering: Synthesis and Opportunities](http://arxiv.org/abs/2505.20099v1), 2025-05-26

(14) [S2LPP: Small-to-Large Prompt Prediction across LLMs](http://arxiv.org/abs/2505.20097v1), 2025-05-26

(15) [Multi-Domain Explainability of Preferences](http://arxiv.org/abs/2505.20088v1), 2025-05-26

(16) [Safety Through Reasoning: An Empirical Study of Reasoning Guardrail Models](http://arxiv.org/abs/2505.20087v1), 2025-05-26

(17) [Inference-time Alignment in Continuous Space](http://arxiv.org/abs/2505.20081v1), 2025-05-26

(18) [Incentivizing Reasoning from Weak Supervision](http://arxiv.org/abs/2505.20072v1), 2025-05-26

(19) [On the Same Page: Dimensions of Perceived Shared Understanding in Human-AI Interaction](http://arxiv.org/abs/2505.20068v1), 2025-05-26

(20) [SafeDPO: A Simple Approach to Direct Preference Optimization with Enhanced Safety](http://arxiv.org/abs/2505.20065v1), 2025-05-26

(21) [Multimodal LLM-Guided Semantic Correction in Text-to-Image Diffusion](http://arxiv.org/abs/2505.20053v1), 2025-05-26

(22) [Grammars of Formal Uncertainty: When to Trust LLMs in Automated Reasoning Tasks](http://arxiv.org/abs/2505.20047v1), 2025-05-26

(23) [REARANK: Reasoning Re-ranking Agent via Reinforcement Learning](http://arxiv.org/abs/2505.20046v1), 2025-05-26

(24) [Uncertainty-Aware Attention Heads: Efficient Unsupervised Uncertainty Quantification for LLMs](http://arxiv.org/abs/2505.20045v1), 2025-05-26

(25) [Correlating instruction-tuning (in multimodal models) with vision-language processing (in the brain)](http://arxiv.org/abs/2505.20029v1), 2025-05-26

(26) [ReasonPlan: Unified Scene Prediction and Decision Reasoning for Closed-loop Autonomous Driving](http://arxiv.org/abs/2505.20024v1), 2025-05-26

(27) [Training LLM-Based Agents with Synthetic Self-Reflected Trajectories and Partial Masking](http://arxiv.org/abs/2505.20023v1), 2025-05-26

(28) [Ontology- and LLM-based Data Harmonization for Federated Learning in Healthcare](http://arxiv.org/abs/2505.20020v1), 2025-05-26

(29) [TTPA: Token-level Tool-use Preference Alignment Training Framework with Fine-grained Evaluation](http://arxiv.org/abs/2505.20016v1), 2025-05-26

(30) [Does Rationale Quality Matter? Enhancing Mental Disorder Detection via Selective Reasoning Distillation](http://arxiv.org/abs/2505.20014v1), 2025-05-26

(31) [WebCoT: Enhancing Web Agent Reasoning by Reconstructing Chain-of-Thought in Reflection, Branching, and Rollback](http://arxiv.org/abs/2505.20013v1), 2025-05-26

(32) [TabPFN: One Model to Rule Them All?](http://arxiv.org/abs/2505.20003v1), 2025-05-26

(33) [NEXT: Multi-Grained Mixture of Experts via Text-Modulation for Multi-Modal Object Re-ID](http://arxiv.org/abs/2505.20001v1), 2025-05-26

(34) [Embracing Imperfection: Simulating Students with Diverse Cognitive Levels Using LLM-based Agents](http://arxiv.org/abs/2505.19997v1), 2025-05-26

(35) [Automatic Metadata Extraction for Text-to-SQL](http://arxiv.org/abs/2505.19988v1), 2025-05-26

(36) [How Well Do Large Reasoning Models Translate? A Comprehensive Evaluation for Multi-Domain Machine Translation](http://arxiv.org/abs/2505.19987v1), 2025-05-26

(37) [DeepDialogue: A Multi-Turn Emotionally-Rich Spoken Dialogue Dataset](http://arxiv.org/abs/2505.19978v1), 2025-05-26

(38) [DFIR-Metric: A Benchmark Dataset for Evaluating Large Language Models in Digital Forensics and Incident Response](http://arxiv.org/abs/2505.19973v1), 2025-05-26

(39) [CP-Router: An Uncertainty-Aware Router Between LLM and LRM](http://arxiv.org/abs/2505.19970v1), 2025-05-26

(40) [Learning to Select In-Context Demonstration Preferred by Large Language Model](http://arxiv.org/abs/2505.19966v1), 2025-05-26

(41) [Adaptive Location Hierarchy Learning for Long-Tailed Mobility Prediction](http://arxiv.org/abs/2505.19965v1), 2025-05-26

(42) [The Limits of Preference Data for Post-Training](http://arxiv.org/abs/2505.19964v1), 2025-05-26

(43) [MiniLongBench: The Low-cost Long Context Understanding Benchmark for Large Language Models](http://arxiv.org/abs/2505.19959v1), 2025-05-26

(44) [DCG-SQL: Enhancing In-Context Learning for Text-to-SQL with Deep Contextual Schema Link Graph](http://arxiv.org/abs/2505.19956v1), 2025-05-26

(45) [MLR-Bench: Evaluating AI Agents on Open-Ended Machine Learning Research](http://arxiv.org/abs/2505.19955v1), 2025-05-26

(46) [An Explainable Diagnostic Framework for Neurodegenerative Dementias via Reinforcement-Optimized LLM Reasoning](http://arxiv.org/abs/2505.19954v1), 2025-05-26

(47) [Multimodal Reasoning Agent for Zero-Shot Composed Image Retrieval](http://arxiv.org/abs/2505.19952v1), 2025-05-26

(48) [Which Data Attributes Stimulate Math and Code Reasoning? An Investigation via Influence Functions](http://arxiv.org/abs/2505.19949v1), 2025-05-26

(49) [Dynamically Learned Test-Time Model Routing in Language Model Zoos with Service Level Guarantees](http://arxiv.org/abs/2505.19947v1), 2025-05-26

(50) [ALAS: Measuring Latent Speech-Text Alignment For Spoken Language Understanding In Multimodal LLMs](http://arxiv.org/abs/2505.19937v1), 2025-05-26

(51) [Subtle Risks, Critical Failures: A Framework for Diagnosing Physical Safety of LLMs for Embodied Decision Making](http://arxiv.org/abs/2505.19933v1), 2025-05-26

(52) [TCP: a Benchmark for Temporal Constraint-Based Planning](http://arxiv.org/abs/2505.19927v1), 2025-05-26

(53) [Enigmata: Scaling Logical Reasoning in Large Language Models with Synthetic Verifiable Puzzles](http://arxiv.org/abs/2505.19914v1), 2025-05-26

(54) [APE: A Data-Centric Benchmark for Efficient LLM Adaptation in Text Summarization](http://arxiv.org/abs/2505.19912v1), 2025-05-26

(55) [EMAC+: Embodied Multimodal Agent for Collaborative Planning with VLM+LLM](http://arxiv.org/abs/2505.19905v1), 2025-05-26

(56) [Dynamic-I2V: Exploring Image-to-Video Generaion Models via Multimodal LLM](http://arxiv.org/abs/2505.19901v1), 2025-05-26

(57) [ScienceBoard: Evaluating Multimodal Autonomous Agents in Realistic Scientific Workflows](http://arxiv.org/abs/2505.19897v1), 2025-05-26

(58) [Large Language Models as Autonomous Spacecraft Operators in Kerbal Space Program](http://arxiv.org/abs/2505.19896v1), 2025-05-26

(59) [ESLM: Risk-Averse Selective Language Modeling for Efficient Pretraining](http://arxiv.org/abs/2505.19893v1), 2025-05-26

(60) [Unifying Multimodal Large Language Model Capabilities and Modalities via Model Merging](http://arxiv.org/abs/2505.19892v1), 2025-05-26

(61) [Deconstructing Obfuscation: A four-dimensional framework for evaluating Large Language Models assembly code deobfuscation capabilities](http://arxiv.org/abs/2505.19887v1), 2025-05-26

(62) [Vad-R1: Towards Video Anomaly Reasoning via Perception-to-Cognition Chain-of-Thought](http://arxiv.org/abs/2505.19877v1), 2025-05-26

(63) [HS-STAR: Hierarchical Sampling for Self-Taught Reasoners via Difficulty Estimation and Budget Reallocation](http://arxiv.org/abs/2505.19866v1), 2025-05-26

(64) [CPA-RAG:Covert Poisoning Attacks on Retrieval-Augmented Generation in Large Language Models](http://arxiv.org/abs/2505.19864v1), 2025-05-26

(65) [Editing as Unlearning: Are Knowledge Editing Methods Strong Baselines for Large Language Model Unlearning?](http://arxiv.org/abs/2505.19855v1), 2025-05-26

(66) [Beyond Specialization: Benchmarking LLMs for Transliteration of Indian Languages](http://arxiv.org/abs/2505.19851v1), 2025-05-26

(67) [Improving Multilingual Math Reasoning for African Languages](http://arxiv.org/abs/2505.19848v1), 2025-05-26

(68) [FoodTaxo: Generating Food Taxonomies with Large Language Models](http://arxiv.org/abs/2505.19838v1), 2025-05-26

(69) [SecVulEval: Benchmarking LLMs for Real-World C/C++ Vulnerability Detection](http://arxiv.org/abs/2505.19828v1), 2025-05-26

(70) [FinLoRA: Benchmarking LoRA Methods for Fine-Tuning LLMs on Financial Datasets](http://arxiv.org/abs/2505.19819v1), 2025-05-26

(71) [Deciphering Trajectory-Aided LLM Reasoning: An Optimization Perspective](http://arxiv.org/abs/2505.19815v1), 2025-05-26

(72) [Efficient Multi-modal Long Context Learning for Training-free Adaptation](http://arxiv.org/abs/2505.19812v1), 2025-05-26

(73) [Exploring Consciousness in LLMs: A Systematic Survey of Theories, Implementations, and Frontier Risks](http://arxiv.org/abs/2505.19806v1), 2025-05-26

(74) [Compliance-to-Code: Enhancing Financial Compliance Checking via Code Generation](http://arxiv.org/abs/2505.19804v1), 2025-05-26

(75) [Integrating emotional intelligence, memory architecture, and gestures to achieve empathetic humanoid robot interaction in an educational setting](http://arxiv.org/abs/2505.19803v1), 2025-05-26

(76) [MOLE: Metadata Extraction and Validation in Scientific Papers Using LLMs](http://arxiv.org/abs/2505.19800v1), 2025-05-26

(77) [What Can RL Bring to VLA Generalization? An Empirical Study](http://arxiv.org/abs/2505.19789v1), 2025-05-26

(78) [Done Is Better than Perfect: Unlocking Efficient Reasoning by Structured Multi-Turn Decomposition](http://arxiv.org/abs/2505.19788v1), 2025-05-26

(79) [Analyzing Political Bias in LLMs via Target-Oriented Sentiment Classification](http://arxiv.org/abs/2505.19776v1), 2025-05-26

(80) [DuRep: Dual-Mode Speech Representation Learning via ASR-Aware Distillation](http://arxiv.org/abs/2505.19774v1), 2025-05-26

(81) [What Really Matters in Many-Shot Attacks? An Empirical Study of Long-Context Vulnerabilities in LLMs](http://arxiv.org/abs/2505.19773v1), 2025-05-26

(82) [SGM: A Framework for Building Specification-Guided Moderation Filters](http://arxiv.org/abs/2505.19766v1), 2025-05-26

(83) [Agentic Predictor: Performance Prediction for Agentic Workflows via Multi-View Encoding](http://arxiv.org/abs/2505.19764v1), 2025-05-26

(84) [Divide and Conquer: Grounding LLMs as Efficient Decision-Making Agents via Offline Hierarchical Reinforcement Learning](http://arxiv.org/abs/2505.19761v1), 2025-05-26

(85) [NeuSym-RAG: Hybrid Neural Symbolic Retrieval with Multiview Structuring for PDF Question Answering](http://arxiv.org/abs/2505.19754v1), 2025-05-26

(86) [Token-level Accept or Reject: A Micro Alignment Approach for Large Language Models](http://arxiv.org/abs/2505.19743v1), 2025-05-26

(87) [ReChisel: Effective Automatic Chisel Code Generation by LLM with Reflection](http://arxiv.org/abs/2505.19734v1), 2025-05-26

(88) [Accelerating Nash Learning from Human Feedback via Mirror Prox](http://arxiv.org/abs/2505.19731v1), 2025-05-26

(89) [Distilling Closed-Source LLM's Knowledge for Locally Stable and Economic Biomedical Entity Linking](http://arxiv.org/abs/2505.19722v1), 2025-05-26

(90) [MT$^{3}$: Scaling MLLM-based Text Image Machine Translation via Multi-Task Reinforcement Learning](http://arxiv.org/abs/2505.19714v1), 2025-05-26

(91) [CAD-Coder: Text-to-CAD Generation with Chain-of-Thought and Geometric Reward](http://arxiv.org/abs/2505.19713v1), 2025-05-26

(92) [MLLM-Guided VLM Fine-Tuning with Joint Inference for Zero-Shot Composed Image Retrieval](http://arxiv.org/abs/2505.19707v1), 2025-05-26

(93) [Error Typing for Smarter Rewards: Improving Process Reward Models with Error-Aware Hierarchical Supervision](http://arxiv.org/abs/2505.19706v1), 2025-05-26

(94) [Point-RFT: Improving Multimodal Reasoning with Visually Grounded Reinforcement Finetuning](http://arxiv.org/abs/2505.19702v1), 2025-05-26

(95) [Leveraging Importance Sampling to Detach Alignment Modules from Large Language Models](http://arxiv.org/abs/2505.19700v1), 2025-05-26

(96) [VisCRA: A Visual Chain Reasoning Attack for Jailbreaking Multimodal Large Language Models](http://arxiv.org/abs/2505.19684v1), 2025-05-26

(97) [Large Language Models for Planning: A Comprehensive and Systematic Survey](http://arxiv.org/abs/2505.19683v1), 2025-05-26

(98) [Deep Actor-Critics with Tight Risk Certificates](http://arxiv.org/abs/2505.19682v1), 2025-05-26

(99) [Large Language Models' Reasoning Stalls: An Investigation into the Capabilities of Frontier Models](http://arxiv.org/abs/2505.19676v1), 2025-05-26

(100) [Calibrating Pre-trained Language Classifiers on LLM-generated Noisy Labels via Iterative Refinement](http://arxiv.org/abs/2505.19675v1), 2025-05-26
